showkase-processor/src/main/java/com/airbnb/android/showkase/processor/writer/ShowkaseBrowserWriter.kt

```diff
@@ -79,15 +79,15 @@ internal class ShowkaseBrowserWriter(private val processingEnv: ProcessingEnviro
                     require(withParameterMetadata is ShowkaseMetadata.Component)
                     addLineBreak()
                     add(
-                        "%T().values.iterator().forEach { previewParam -> ",
+                        "%T().values.iterator().asSequence().forEachIndexed { index, previewParam -> ",
```

Comment:
this index will be used when generating the key below

Reply:
So are the values runtime-dynamic? Just curious is this key could be generated statically

Reply:
Interesting. Will look into it..

Reply:
I looked at it and don't think it will save us anything because I'll have to use reflection to generate this statistically. Are the benefits of doing that significantly better? @BenSchwab